pkg/report/report.go

```diff
@@ -182,6 +182,18 @@ func behaviorRisk(ns string, rule string, tags []string) int {
 	for _, tag := range tags {
 		if r, ok := levels[tag]; ok {
 			risk = r
+			break
+		}
+	}
+
+	// https://github.com/dustinkirkland/byobu/blob/cd253f0229b68d3c2cffa8d862309c6ea7019d94/usr/bin/vigpg#L25
```

Comment:
Let's keep all rule-specific behaviors in YARA. Here's how I would probably set out to solve it:

- Exclude this expression (effectively mkstemp(3)-style use) from the current YARA rule
- Add a new rule specifically for mkstemp(3) usage of /dev/shm with a lower priority.


Probably the best way to do this is up this regexp as an exclusion for the existing rule, and then add another rule that specifically matches

Reply:
Cool -- that makes sense!

Reply:
Updated in [`f1b2733` (#302)](https://github.com/chainguard-dev/bincapz/pull/302/commits/f1b27330e8325dff10f09ff410902b58d4cf15d2).